Misc doc changes

kianmeng opened the pull request:
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

kianmeng commented:
@bglusman Should I resolve the conflict and resubmit the PR again?

bglusman commented:
@kianmeng oh thank you! I hadn't seen this when I bumped versions a few weeks ago, much appreciated!

bglusman commented:
Hmm also hand't noticed its now failing lots of tests on CI!  😢 Feel free to take a crack at fixing those also or I'll try and get to soon if you don't

kianmeng committed:
Misc doc changes

Besides other documentation changes, this commit includes all relevant
documents to the generated HTML doc for HexDocs.pm and leverages on
latest features of ExDoc.

kianmeng commented:
@bglusman Done. Please check.

Also added another PR for GitHub Actions, https://github.com/bglusman/live_dashboard_history/pull/6.

bglusman committed:
Merge branch 'master' into misc-doc-changes

bglusman commented:
Hmm, sorry I didn't look at this in 2 years 🙃 in my copious free time I'll try and figure out whats going on, but, also, apologies I'll try and be more responsive next time.

kianmeng commented:
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:
